text/simple_extensions_schema.yaml

```diff
@@ -10,6 +10,7 @@ properties:
     items:
       type: object
       additionalProperties: false
+      required: [name]
```

Comment:
Wouldn't `structure` also be required?

Reply:
Got it.  I found this paragraph on the [site](https://substrait.io/types/type_classes/#user-defined-types):

> The structure field is optional. If not specified, the type class is considered to be fully opaque. This implies that a systems without built-in support for the type cannot manipulate values in any way, including moving and cloning. This may be useful for exotic, context-sensitive types, such as raw pointers or identifiers that cannot be cloned.

Sorry for the goose chase